README.md

```diff
@@ -1,7 +1,87 @@
 # transmission-heterogeneity-results
 
-This repository holds the results for the [branching process model](https://github.com/SABS-R3-Epidemiology/branchpro).
+This repository holds the results for the [branching process model](https://github.com/SABS-R3-Epidemiology/branchpro). This includes generating serial interval for COVID-19, parsing data from raw COVID-19 cases dataand reproducing figures from the paper.
 
-## Notebook upload procedure
-1. Make sure the filename of the notebook includes figure number and a description, e.g. fig1-schematic.
-2. Make sure the notebooks save their figures to the figures directory, with consistent filenames matching the paper.
+## Reproduce all results (serial interval, parsed data for COVID-19 cases, figures)
+To reproduce all results, including intermediate files:
+```bash
+make
+```
+
+To delete all files created:
+```bash
+make clean
+```
+
+
+## Serial interval
+To generate serial interval for COVID-19:
+```bash
+make serial_interval
+```
+
+To delete file of serial interval for COVID-19:
+```bash
+make clean_serial_interval
+```
+
+
+## COVID-19 cases
+To parse all data from raw COVID-19 cases:
+```bash
+make data
+```
+
+To parse specific data:
+```bash
+make X_data
+```
+where `X` is the name of region. A list of commands to parse the data are given below.
+
+To delete files of parsed data:
+```bash
+make clean_data
+```
+### List of commands to parse data
+1. `australia_data` - to parse Australia's data
+2. `hawaii_data` - to parse Hawaii's data
+3. `new_zealand_data` - to parse New Zealand's data
+4. `ontario_data` - to parse Ontario's data
+
+For example, to parse Australia's data:
+```bash
+make australia_data
+```
+
+
+## Figures
+To reproduce all figures:
```

Comment:
Can you add a statement here saying that running the figures requires that the appropriate data and serial intervals have already been generated as described above

Reply:
Then I'll link #7 here.